modules/pubmaticBidAdapter.js

```diff
@@ -220,26 +172,37 @@ function _parseAdSlot(bid) {
   }
   // check if size is mentioned in sizes array. in that case do not check for @ in adslot
   splits = slot.split('@');
-  if (splits.length != 2) {
-    if (!(sizesArrayExists)) {
-      utils.logWarn(LOG_WARN_PREFIX + 'AdSlot Error: adSlot not in required format');
+  /* if (splits.length != 2) {
+    if (!sizesArrayExists) {
+      utils.logWarn('AdSlot Error: adSlot not in required format');
       return;
     }
-  }
+  } */
   bid.params.adUnit = splits[0];
-  if (splits.length > 1) { // i.e size is specified in adslot, so consider that and ignore sizes array
+  if (splits.length > 1) {
+    // i.e size is specified in adslot, so consider that and ignore sizes array
     splits = splits[1].split('x');
     if (splits.length != 2) {
       utils.logWarn(LOG_WARN_PREFIX + 'AdSlot Error: adSlot not in required format');
       return;
     }
     bid.params.width = parseInt(splits[0]);
     bid.params.height = parseInt(splits[1]);
-    delete bid.sizes;
-  } else if (sizesArrayExists) {
+    // delete bid.sizes;
```

Comment:
can we remove commented code?

Reply:
done

Reply:
Hello @pm-manasi-moghe ,
Let's keep this comment unresolved until we merge the changes into this branch.